docs/source/usage.rst

```diff
@@ -74,3 +74,58 @@ Positions:
   - ``H``: OK
   - ``7``: failure to charge HV capacitor
 
+BLE - Bluetooth® Low Energy
+###########################
+
+BLE Heart Rate Service
+----------------------
+
+The MultiGeiger, starting from firmware version 1.15.0-dev, provides a Bluetooth® Low Energy (BLE) service to collect the count rate via a `GATT Heart Rate Service <https://www.bluetooth.com/specifications/gatt/services/>`_ (Service UUID 0x180D). The following characteristics are used:
```

Comment:
guess you do not need to refer to a version number here.

the docs are version, so if somebody is looking into V1.14.0 release docs, they won't see any BLE stuff.
and if they look into V1.15+ docs, they will.

Reply:
s/collect/provide/ ?

Reply:
Okay, can remove the version. However we provide the service, so the data can be collected by another device :)

Reply:
if you continue the sentence with "so that data can be collected by another device", it won't be misunderstandable.

Reply:
Did that indeed (in a way :)